notify/impl.go

```diff
@@ -474,17 +488,41 @@ func (n *PagerDuty) Notify(ctx context.Context, as ...*types.Alert) (bool, error
 		details[k] = tmpl(v)
 	}
 
-	msg := &pagerDutyMessage{
-		ServiceKey:  tmpl(string(n.conf.ServiceKey)),
-		EventType:   eventType,
-		IncidentKey: hashKey(key),
-		Description: tmpl(n.conf.Description),
-		Details:     details,
+	msg := &pagerDutyMessage{}
+
+	if n.conf.ServiceKey != "" {
+		msg.ServiceKey = tmpl(string(n.conf.ServiceKey))
```

Comment:
Add some logging here saying that the v1 support is deprecated. Maybe add the user to the pagerduty faq where they day the date that they still stop supporting it.

Reply:
:+1: